docs/contribution/35-guidelines/20-privacy.mdx

```diff
@@ -0,0 +1,110 @@
+---
+title: Privacy and GDPR Policy for Extensions in the mStudio marketplace
+sidebar_label: Privacy policy
+---
+
+This privacy and GDPR policy establishes the requirements that contributors must follow to ensure compliance with data protection regulations, including GDPR, when developing and distributing extensions in the mStudio Marketplace. The goal is to protect user privacy and maintain transparency in data handling.
+
+## Data Deletion Policy
+
+### Deletion of Data on Uninstallation
+
+When a user uninstalls an _Extension Instance_, any data stored or processed by the extension related to that extension instance must be securely deleted. This includes:
+
+- User-generated data stored locally or remotely by the extension.
```

Comment:
The "remotely" part of this depends on whether we want to allow extensions to "eject" themselves (i.e. leave any managed resources that they created to be managed by the user themselves, when the extension is installed).

Reply:
This is a delicate topic. We still do not have a solution to enable contributors to "eject" their extension instance. 
This is still a trade off situation between lifetime of access tokens and possibility of cleaning up mStudio resources.

Reply:
Isn't "ejecting" the default right now? Since the access tokens are revoked before the `ExtensionInstanceRemovedFromContext` webhook is invoked, there's no possibility to clean up anything (at least, any mStudio resources), anyway.

Reply:
yeah wrong terminology. i meant we currently offer no way of cleaning up the mStudio resources created by the extension, but this topic possibly will get some traction shortly.